__tests__/auth.test.js

```diff
@@ -17,8 +19,39 @@ describe('tc-ghoauth routes', () => {
     return agent;
   };
 
-  it('is a blank test', async () => {
-    const agent = await newAgent();
-    expect(agent).toBeTruthy();
+  it('creates a cookie jar', async () => {
+    const { jar } = await newAgent();
+    const cookieJar = jar;
+    expect(cookieJar).toBeTruthy();
+  });
+
+  it('redirects to githubs OAuth', async () => {
+    const res = await request.agent(app).get('/api/v1/github/login');
+    expect(res.header.location).toMatch(
+      /https:\/\/github.com\/login\/oauth\/authorize\?client_id=[\w\d]+&scope=user&redirect_uri=[\w\d]/i
+    );
+  });
+
+  it('should login and redirect users to /api/v1/github/dashboard', async () => {
```

Comment:
No dashboard route required for this deliverable. The redirect should go directly to /api/v1/posts.